nibabel/nicom/tests/test_csareader.py

```diff
@@ -14,6 +15,8 @@
 
 from .test_dicomwrappers import (have_dicom, dicom_test,
                                  IO_DATA_PATH, DATA, DATA_FILE)
+if have_dicom:
+    from .test_dicomwrappers import pydicom
```

Comment:
Additionally, better to use `skipIf` decorator on the test: `skipif(not have_dicom)`